README.md

```diff
@@ -1,29 +1,310 @@
-# OpenFeature SDK for Rust
 
-[![Project Status: WIP – Initial development is in progress, but there has not yet been a stable, usable release suitable for the public.](https://www.repostatus.org/badges/latest/wip.svg)](https://www.repostatus.org/#wip)
+<!-- TODO: This is a template SDK README.md.
+It's structured to fit nicely into openfeature.dev, which is configured to include SDK READMEs.
+Complete all the TODOs applicable for your implementation and then create an issue in https://github.com/open-feature/openfeature.dev
+-->
 
-This is the rust implementation of [OpenFeature](https://openfeature.dev), a vendor-agnostic abstraction library for evaluating feature flags.
+<!-- markdownlint-disable MD033 -->
+<!-- x-hide-in-docs-start -->
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/open-feature/community/0e23508c163a6a1ac8c0ced3e4bd78faafe627c7/assets/logo/horizontal/white/openfeature-horizontal-white.svg" />
+    <img align="center" alt="OpenFeature Logo" src="https://raw.githubusercontent.com/open-feature/community/0e23508c163a6a1ac8c0ced3e4bd78faafe627c7/assets/logo/horizontal/black/openfeature-horizontal-black.svg" />
+  </picture>
+</p>
 
-We support multiple data types for flags (numbers, strings, booleans, objects) as well as hooks, which can alter the lifecycle of a flag evaluation.
+<h2 align="center">OpenFeature Rust SDK</h2>
 
-## Installation
+<!-- x-hide-in-docs-end -->
+<!-- The 'github-badges' class is used in the docs -->
+<p align="center" class="github-badges">
+  <a href="https://github.com/open-feature/spec/tree/v0.7.0">
+    <img alt="Specification" src="https://img.shields.io/static/v1?label=specification&message=v0.7.0&color=yellow&style=for-the-badge" />
+  </a>
+  <!-- TODO: update the Release Please config to include the readme -->
+  <!-- x-release-please-start-version -->
+
+<!-- TODO: update with your SDK repo and the latest release version
+  <a href="https://github.com/open-feature/my-sdk/releases/tag/v0.0.1">
+    <img alt="Release" src="https://img.shields.io/static/v1?label=release&message=v0.0.1&color=blue&style=for-the-badge" />
+  </a>  
+-->
+
+  <!-- x-release-please-end -->
+  <br/>
+  <a href="https://bestpractices.coreinfrastructure.org/projects/6601">
```

Comment:
This should be removed until we have a CII best practices entry for Rust.

Reply:
Commented out. Thank you.